components/admin_console/admin_console.jsx

```diff
@@ -382,27 +382,35 @@ export default class AdminConsole extends React.Component {
                         )}
                         />
                         <Route
-                            path={`${this.props.match.url}/advanced`}
+                            path={`${this.props.match.url}/compliance`}
                             render={(props) => (
                                 <Switch>
                                     <SCRoute
-                                        path={`${props.match.url}/rate`}
-                                        component={RateSettings}
+                                        path={`${props.match.url}/dataretention`}
```

Comment:
nit: `dataretention` vs. `message_export`. Do we have a convention for when to use an underscore to break up words in these kinds of links?

Reply:
Good catch, not directly related with my PR, but is a good time to fix it. Looks like every other string is using snake case for this, so have sens to me to change it to data_retention.

Reply:
changed :)